tests/check_files/test_filesystem_integrity/data/warning_list.txt

```diff
@@ -0,0 +1,31 @@
+/etc/group
```

Comment:
You are duplicating this file.

You have to decide if add this file to the framework data files, or as testing data file,

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/2437/commits/24801f68dd74da1dce8fb303004cbb0c65e79a25

I think it is better to have it with all the check-files tests